devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -199,7 +199,7 @@ public void startDev() {
                     System.err.println("Port 5005 in use, not starting in debug mode");
                 } catch (IOException e) {
                     args.add("-Xdebug");
-                    args.add("-Xrunjdwp:transport=dt_socket,address=5005,server=y,suspend=" + debugSuspend);
+                    args.add("-Xrunjdwp:transport=dt_socket,address=0.0.0.0:5005,server=y,suspend=" + debugSuspend);
                 }
             } else if (debug.toLowerCase().equals("client")) {
                 args.add("-Xdebug");
```

Comment:
The line below uses `localhost`

Reply:
Here localhost is used too: https://github.com/quarkusio/quarkus/pull/7890/files#diff-72fac779acece3ea68a6666ee0060ad5R209

Reply:
Yes, the code before had `localhost` there too, therefore I didn't want to change the behavior. But agree, we could think about if it's not more consistent to allow remote connections everywhere

Reply:
IMHO it should all be `0.0.0.0` unless proven wrong :wink: